BookEcommerce.Models/DAL/Repositories/RoleRepository.cs

```diff
@@ -25,9 +25,9 @@ public async Task AddToRole(ApplicationUser ApplicationUser, string Role)
 
         public async Task<bool> CheckRole(string RoleName)
         {
-            var RoleExist = await this.roleManager.RoleExistsAsync(RoleName);
-            if (!RoleExist) return false;
-            else return true;
+            var roleExist = await this.roleManager.RoleExistsAsync(RoleName);
+            var existed = !roleExist ? false : true;
+            return existed;
```

Comment:
not a good code here. You can simply
return roleExist;

Reply:
resolved